bftengine/tests/simpleStorage/FileStorage.cpp

```diff
@@ -133,7 +133,10 @@ bool FileStorage::initMaxSizeOfObjects(ObjectDesc *metadataObjectsArray, uint32_
   const size_t maxFileSize = fileMetadataSize + maxObjectsSize;
   LOG_INFO(logger_, "FileStorage::initMaxSizeOfObjects Maximum size of objects is "
       << maxObjectsSize << ", file size is " << maxFileSize);
-  ftruncate(fileno(dataStream_), maxFileSize);
+  if (ftruncate(fileno(dataStream_), maxFileSize)) {
+    LOG_ERROR(logger_, "Failed to truncate file: " << fileName_);
+    assert(true);
```

Comment:
I know there have been a few different assertion libraries used around here. Maybe I'm thinking of the wrong one, but isn't `assert(true)` a no-op?

Reply:
Whoops! Great catch! Yes, this should be `assert(false)`.